src/unittests/f32_interval.spec.ts

```diff
@@ -457,3 +458,51 @@ g.test('ulpInterval')
       `ulpInterval(${value}, ${num_ulp}) returned ${got}. Expected ${expected}`
     );
   });
+
+interface PointToIntervalCase {
```

Comment:
This is the only case type in this file. Could we call it `Case` instead?

Reply:
There will be future `BinaryToIntervalCase` and `TernaryToIntervalCase` as I get to builtins that need them.